SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorLayoutDetails.cpp

```diff
@@ -245,13 +277,33 @@ bool FSpatialGDKEditorLayoutDetails::TryPushCommandLineArgsToDevice(const FStrin
 	return true;
 }
 
+namespace
+{
+	static FString GetAdbExePath()
```

Comment:
I will suggest writing the function like this: 
static bool GetAdbExePath(FString& AdbExe)

Reply:
I understand the reason to use an output parameter (e.g. by reference) for preventing a copy, and the bool allows us to skip calling IsEmpty()..

I would counter that return-value optimization and copy-elision are common in most modern compilers:
https://stackoverflow.com/questions/12953127/what-are-copy-elision-and-return-value-optimization

Finally in this case I'll err on following the existing pattern found commonly in the code, which to my current understanding is just to make a function that returns FString.